homeassistant/components/image/__init__.py

```diff
@@ -348,7 +348,7 @@ async def _write_frame() -> bool:
         # While this results in additional bandwidth usage,
         # given the low frequency of image updates, it is acceptable.
         frame.extend(frame)
-        await response.write(frame)
+        await response.write(frame)  # type: ignore[arg-type]
```

Comment:
Fixed upstream in https://github.com/aio-libs/aiohttp/pull/10154
Will be included in the next aiohttp release.

FYI @bdraco